extensions/wikia/AdEngine/js/provider/gpt/adDetect.js

```diff
@@ -233,6 +234,11 @@ define('ext.wikia.adEngine.provider.gpt.adDetect', [
 			return callAdCallback();
 		}
 
+		if (adType === 'collapse') {
+			slotTweaker.hide(slotname);
```

Comment:
We discussed it with @gabrys and it would make more sense if we pass the "request" to collapse the slot outside the `adDetect` module. We could ask gptHelper to collapse the slot or the direct provider.